swo/mpt/cli/core/plugins.py

```diff
@@ -0,0 +1,19 @@
+from importlib.metadata import EntryPoint, entry_points
+
+from typer import Typer
+
+PLUGINS_PACKAGE = "swo.mpt.cli.plugins"
+
+
+def list_entrypoints() -> list[EntryPoint]:  # pragma: no cover
+    entrypoints = entry_points().select(group=PLUGINS_PACKAGE)
+    if not entrypoints:
+        return []
+
+    return entrypoints
+
+
+def load_plugins(app: Typer):
+    for entrypoint in list_entrypoints():
+        plugin = entrypoint.load()
+        app.add_typer(plugin.app, name=entrypoint.name)
```

Comment:
IMO we shouldn't force the implementor of a plugin to call the app `app`.
So probably it's better to make the implementor to explicitly point to the app instance like:
`cli_adobe_plugin.app:app` instead of pointing to the module like `cli_adobe_plugin.app`.

Reply:
Agre